addon/initializers/route-alias.js

```diff
@@ -68,13 +68,14 @@ function patchRoute(lookup) {
       };
 
       // Figure out how many routes we created.
-      let originalLength = this.matches.length;
+      let originalLength = [].concat(...this.matches).length;
```

Comment:
if we're concerned about the spread syntax, i'll commit another change for flattening the arrays.